resources/views/includes/elements/payment-options.blade.php

```diff
@@ -0,0 +1,37 @@
+<div class="container mt-4 p-5">
+    <div class="card mt-4 col-md-12">
+        <div class="row mb-4 no-gutters">
+            <h3 class="lead p-heading p-heading--single">Friends individual</h3>
+            <hr /> 
+            <div class="col-md-4 border d-flex rounded text-center p-container p-container--single">
+                <span>Friends individual</span>
+                <figure class="fluid">
+                    <img class="img-fluid p-image p-image--single" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0iY3VycmVudENvbG9yIiA+CiAgPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBkPSJNNy41IDZhNC41IDQuNSAwIDExOSAwIDQuNSA0LjUgMCAwMS05IDB6TTMuNzUxIDIwLjEwNWE4LjI1IDguMjUgMCAwMTE2LjQ5OCAwIC43NS43NSAwIDAxLS40MzcuNjk1QTE4LjY4MyAxOC42ODMgMCAwMTEyIDIyLjVjLTIuNzg2IDAtNS40MzMtLjYwOC03LjgxMi0xLjdhLjc1Ljc1IDAgMDEtLjQzNy0uNjk1eiIgY2xpcC1ydWxlPSJldmVub2RkIiAvPgo8L3N2Zz4KCg==" alt="user icon" />  
```

Comment:
Can you please add a comment block explaining where you got the SVG icons from? Just incase someone needs to add a new one (or modify this existing one).

Something like this should work:

```
{{--
    The base64 encoded icons come from: [url-here]
--}}
```